doc/source/structures/misc/config.rst

```diff
@@ -127,11 +127,15 @@ Configuration of kOS
 
     Configures the ``EnableSafeMode`` setting.
 
+.. highlight:: none
+
     If true, then it enables the following error messages::
 
         Tried to push NaN into the stack.
         Tried to push Infinity into the stack.
 
+.. highlight:: kerboscript
+
```

Comment:
Again, I got no such complaints in my run.  I wonder if there's a sphinx version difference.